getting-started/quick-start.md

```diff
@@ -1,54 +1,45 @@
-# Zero to Deploy
+# Quick Start - Zero to Deploy
 
-# Getting Started with Amber
-This getting started guide will help you get a full stack web application running in just a few minutes by leveraging amber’s code generators.
-
-After we have a working application, then we will walk through the code that is making it all happen and code some enhancements along the way. (Additional guides are pending)
+This quick start guide will help you get a full stack web application running in just a few minutes by leveraging amber’s code generators.
 
 Let’s get started!
 
 ## Install dependencies
+
 If you already have `crystal`, `amber`, and a database installed, you can skip this step.
 
-### Install `crystal`
+### Install `crystal` and `amber`
+
+Instructions for OS X using homebrew and Debian/Ubuntu are below. See full installation instructions [here](https://crystal-lang.org/docs/installation/) for RedHat & CentOS, ArchLinux & Derivatives, and more complete instructions.
 
 #### OS X with homebrew
-_You can skip this step if installing amber with homebrew, it installs crystal_
+
+Installing amber with homebrew also installs crystal
  ```
-brew install crystal-lang
+brew install amberframework/amber/amber
```

Comment:
This will only work if you have already installed the `tap`

```
brew tap amberframework/amber
brew install amber
```

Reply:
I thought @elorest said the tap was optional? Is there an easy way to confirm this?

Reply:
`brew install amberframework/amber/amber` automatically taps amberframework/amber.